examples/ui-demo/src/components/preview/MobileSplashPage.tsx

```diff
@@ -32,7 +32,7 @@ export function MobileSplashPage() {
             onchain
           </span>
         </h3>
-        <p className="text-base font-normal tracking-tight text-center text-demo-fg-secondary">
+        <p className="text-base font-normal tracking-tight text-center text-fg-secondary">
```

Comment:
We might want to even update the color of Try it on line 60 to white in dark mode.

Reply:
The button styling (`akui-btn-primary`) across the components uses dark text (inverted fg primary color, to be expact) on the primary button in dark mode:

![Screenshot 2025-03-18 at 1 03 28 PM](https://github.com/user-attachments/assets/02af9f11-3200-4054-b4c8-c4e40504d354)

```ts
  ".akui-btn-secondary": {
    "@apply bg-btn-secondary text-fg-primary": {},
  },
```